sdk/iothub/azure-iot-deviceprovisioning/azure/iot/deviceprovisioning/_api_version.py

```diff
@@ -0,0 +1,15 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from enum import Enum
+
+from azure.core import CaseInsensitiveEnumMeta
+
+
+class ApiVersion(str, Enum, metaclass=CaseInsensitiveEnumMeta):
+    V2021_10_01 = "2021-10-01"
```

Comment:
is this actually the newest one? Apparently for control plane, there was one public api (2022-02-05) that we skipped (cause no one told us and I think it had no new features)

Reply:
It's the latest dataplane API included in the [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/deviceprovisioningservices/data-plane) repo.

For control plane there is an even [newer one](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/stable/2022-12-12) than the one you mentioned.

Reply:
😠